src/Tizen.NUI/src/internal/WebView/WebContextMenu.cs

```diff
@@ -0,0 +1,155 @@
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// It is a class for context menu of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebContextMenu : Disposable
+    {
+        internal WebContextMenu(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
+        {
+        }
+
+        /// This will not be public opened.
+        /// <param name="swigCPtr"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef swigCPtr)
+        {
+            Interop.WebContextMenu.DeleteWebContextMenu(swigCPtr);
+        }
+
+        /// <summary>
+        /// Counts items of the context menu.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint ItemCount
+        {
+            get
+            {
+                return Interop.WebContextMenu.GetItemCount(SwigCPtr);
+            }
+        }
+
+        /// <summary>
+        /// Gets the list of items.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebContextMenuItemList ItemList
+        {
+            get
+            {
+                IntPtr result = Interop.WebContextMenu.GetItemList(SwigCPtr);
+                return new WebContextMenuItemList(result, true);
+            }
+        }
+
+        /// <summary>
+        /// Gets position of the context menu.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Vector2 Position
+        {
+            get
+            {
+                IntPtr result = Interop.WebContextMenu.GetPosition(SwigCPtr);
+                return new Vector2(result, true);
+            }
+        }
+
+        /// <summary>
+        /// Returns the nth item in a context menu.
+        /// <param name="index">The position of the item</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebContextMenuItem GetItemAtIndex(uint index)
+        {
+            IntPtr result = Interop.WebContextMenu.GetItemAt(SwigCPtr, index);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return new WebContextMenuItem(result, true);
+        }
+
+        /// <summary>
+        /// Removes the item from the context menu.
+        /// <param name="item">The item to be removed</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool RemoveItem(WebContextMenuItem item)
+        {
+            bool result = Interop.WebContextMenu.RemoveItem(SwigCPtr, WebContextMenuItem.getCPtr(item));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return result;
+        }
+
+        /// <summary>
+        /// Adds the item to the context menu.
```

Comment:
Can you specify the difference with `AppendItem()`?

Reply:
Done.
I added some words according to function name.

Reply:
I still don't get what the different is. If "AsAction" means it appends an item as an action, how can user specify the action?

Reply:
In both AppendItem and AppendItemAsAction, item can be added as an action according to source code of chromium-efl.
The difference is their parameters.
C-style APIs could have the same function names, while C# could.
So I renamed AppendItemAsAction to AppendItem.